code/go/internal/validator/spec.go

```diff
@@ -127,13 +127,17 @@ func processErrors(errs specerrors.ValidationErrors) specerrors.ValidationErrors
 			new:     "%s: rename \"message\" to \"event.original\" processor requires remove \"message\" processor",
 		},
 		{
-			matcher: regexp.MustCompile(`(processors.[0-9]+.remove.field): processors.[0-9]+.remove.field does not match: "message"`),
+			matcher: regexp.MustCompile(`(processors.[0-9]+.remove.field(.[0-9]+)?): processors.[0-9]+.remove.field(.[0-9]+)? does not match: "message"`),
 			new:     "%s: rename \"message\" to \"event.original\" processor requires remove \"message\" processor",
 		},
 		{
 			matcher: regexp.MustCompile(`(processors.[0-9]+.remove.if): processors.[0-9]+.remove.if does not match: "ctx\.event\?\.original != null"`),
 			new:     "%s: rename \"message\" to \"event.original\" processor requires remove \"message\" processor with if: 'ctx.event?.original != null'",
 		},
+		{
+			matcher: regexp.MustCompile(`(field processors.[0-9]+.remove): (ignore_missing|if) is required`),
+			new:     "%s: rename \"message\" to \"event.original\" processor requires remove \"message\" processor with if: 'ctx.event?.original != null'",
+		},
```

Comment:
This rule is added to allow to ignore the error in pipelines that only have a `remove` processor and it doesn't have the `ignore_missing` or `if` properties. This is the case of the pipeline added here for testing.

Other pipelines are not failing because they use to contain some other remove with these parameters, and if the `if` doesn't contain `'ctx.event?.original != null'`, it is captured by the previous rule above.